test('should fail with timeout', (_done) => {
  // _done() callback is never called
});
